Títols generalitzats

Arnaldopower opened the pull request.

Arnaldopower committed:
literal un fitxer buit

Arnaldopower committed:
Generalitzar en procés-part1

Arnaldopower committed:
Generalitzar titols - part 2

Arnaldopower committed:
Treure els que no es fan servir

Arnaldopower committed:
a

Arnaldopower committed:
Apply formatting changes

Quiracle reviewed (commented):
No fiquis paraules en catala, canvia negreta per bold i negre per black.

Quiracle committed:
Merge branch 'integration' into profile-fix

Arnaldopower committed:
fix: shirt size in black letter and disable visit other people profiles

Arnaldopower committed:
Merge branch 'profile-fix' of https://github.com/LleidaHack/lleidahack-frontend into profile-fix

Arnaldopower committed:
fix: parameters in title component in english

Arnaldopower committed:
Apply formatting changes

Quiracle reviewed (commented):
Arnau encara hi ha moltes coses en catala jsfksdjf 
el camp subrallat, colorLLetra, moltes classes en css i la variable que fas anar pel classname
no se si em deixo algo, revisaho per si a cas

Arnaldopower committed:
fix: Les grans ciutats del país volen més català a l’escola

Alcaldes i regidors reclamen despolititzar el debat i avisen que cal reforçar la llengua fora de les aules

Arnaldopower committed:
Merge branch 'profile-fix' of https://github.com/LleidaHack/lleidahack-frontend into profile-fix

Arnaldopower committed:
centered

Quiracle commented on src/components/TitleGeneralized/TitleGeneralized.js:
```diff
@@ -0,0 +1,43 @@
+  alignText = "center",
+  size,
+  padTop = "5%",
+  margeBot = "0%",
```
marge es catala diria

Quiracle commented on src/components/Profile/profile.js:
```diff
@@ -200,11 +207,13 @@ const ProfileComponent = () => {
-              {hacker && <EditProfile hacker={hacker} />}
+          {isUser && (
+            <div className="espaiEdit">
+              <div className="ajustarEdit">
```
espai i ajustar en catala

Arnaldopower committed:
fix: ¿Por qué la gente no aprende inglés? Porque el inglés se enseña mal. ¡Punto!

Arnaldopower committed:
fix: era la marge dels simpson

Arnaldopower committed:
fix: aixo no ha d'estar aqui

Arnaldopower committed:
fix: actualitzar talla samarreta
